### PR TITLE
Different functions

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Please review these files
-[src/AllRoutes.jsx](https://github.com/Dev-AkshatGupta/piston-notes-maker/blob/12f6f1c8059fa9e40d7bed888d56af94180f9491/src/AllRoutes.jsx)
[Redux/Redux/Reducers]([piston-notes-maker](https://github.com/Dev-AkshatGupta/piston-notes-maker)/[src](https://github.com/Dev-AkshatGupta/piston-notes-maker/tree/main/src)/[Redux](https://github.com/Dev-AkshatGupta/piston-notes-maker/tree/main/src/Redux)/Reducers-Redux/)

Please ignore any kind of new css files else it can be reviewed thanks

### CODE REVIEW

File: src/Redux/Reducers-Redux/notesSlice.js [hunk 1]
```diff
@@ -1,14 +1,343 @@
...
+    const encodedToken = localStorage.getItem("token");
+    try {
+      const { data } = await axios.get(
+        "/api/user",
+        {},
+        {
+          headers: { authorization: encodedToken },
+        }
+      );
+      console.log(data);
```
Comment (dev-prabhat) [inline src/Redux/Reducers-Redux/notesSlice.js:25]: remove console.log

File: src/Redux/Reducers-Redux/notesSlice.js [hunk 2]
```diff
@@ -1,14 +1,343 @@
...
+      return data;
+    } catch (error) {
+      notifyError(error.response.data.errors[0]);
+      console.log(error.response);
```
Comment (dev-prabhat) [inline src/Redux/Reducers-Redux/notesSlice.js:50]: remove console.log

**Review by dev-prabhat — APPROVED** [review]
Good to merge 
Looks good to me

**Review by mikesch803 — APPROVED** [review]
good to merge

**Pull request merged**